deptry/dependency.py

```diff
@@ -45,6 +47,18 @@ def __init__(
     def _get_top_levels(
         self, name: str, distribution: Distribution | None, module_names: Sequence[str] | None
     ) -> set[str]:
+        """
+        Get the top-level module names for a dependency. They are searched for in the following order:
+                1. If `module_names` is defined, simply use those as the top-level modules.
+                2. Otherwise, try to extract the top-level module names from the metadata.
+                3. If metadata extraction fails, fall back to the dependency's name, where `-` is replaced with `_`.
+
+        Args:
+            name (str): The name of the dependency.
+            distribution (Optional[Distribution]): The metadata distribution of the package.
+            module_names (Optional[Sequence[str]]): If this is given, use these as the top-level modules instead of
```

Comment:
```suggestion
            name: The name of the dependency.
            distribution: The metadata distribution of the package.
            module_names: If this is given, use these as the top-level modules instead of
```
Since we already define the types using type annotations, this is redundant, and could lead to a drift if we update type annotations but forget to update the docstrings.